src/service/fake_generator.py

```diff
@@ -0,0 +1,35 @@
+import skimage
+import os
+import random
+
+from model.generator import Generator
+
+
+class FakeGenerator(Generator):
+    dir_path = "../../example/ai_gen_images/"
+    images = []
+
+    def generate(self, output_file_path):
+        """
+           Method that takes one random image of the ai_gen_image directory and puts it in another directory.
+        """
+        self.list_images()
```

Comment:
Instead of creating a class attribute here, I would go with a object attribute (aka set it in the constructor). Then you also do not need the helper method but you can just run the code in the constructor.

Reply:
i understand and agree, but if i runn the same code in the consturctor, i still have to safe it in a class attribute no?

Reply:
Yeah but the self.list_images() method already saves it in class attribute called images.